src/app/data.service.ts

```diff
@@ -64,27 +63,27 @@ export class DataService {
-                    const tempStart = dayjs(meta.result.max_timestamp)
-                        .subtract(1, "year")
-                        .startOf("day")
-                        .subtract(dayjs().utcOffset(), "minute")
-                        .format('YYYY-MM-DDTHH:mm:ss') + 'Z'
+                    // const tempStart = dayjs(meta.result.max_timestamp)
+                    //     .subtract(1, "year")
+                    //     .startOf("day")
+                    //     .subtract(dayjs().utcOffset(), "minute")
+                    //     .format('YYYY-MM-DDTHH:mm:ss') + 'Z'
                     // if URL params are empty then fill it with default values
-                    const queryParams = this.getQueryParamsFromFragments();
-                    const defaults: IQueryParam = {
-                        hashtag: queryParams && queryParams.hashtag ? queryParams.hashtag : this.defaultHashtag,
-                        interval: queryParams && queryParams.interval ? queryParams.interval : this.defaultIntervalValue,
-                        start: queryParams && queryParams.start ? queryParams.start : queryParams && queryParams.hashtag ? meta.result.min_timestamp : tempStart,
-                        end: queryParams && queryParams.end ? queryParams.end : this.maxDate,
-                        countries: queryParams && queryParams.countries ? queryParams.countries : '',
-                        topics: queryParams && queryParams.topics ? queryParams.topics : ''
-                    }
-                    if (queryParams?.fit_to_content !== undefined) {
-                        defaults.fit_to_content = queryParams.fit_to_content
-                    }
-                    this.updateURL(defaults)
+                    // const queryParams = this.getQueryParamsFromFragments();
+                    // console.log('>>> requestMetadata >>> ', queryParams);
+                    // const defaults: IQueryParam = {
+                    //     hashtag: queryParams && queryParams.hashtag ? queryParams.hashtag : this.defaultHashtag,
+                    //     interval: queryParams && queryParams.interval ? queryParams.interval : this.defaultIntervalValue,
+                    //     start: queryParams && queryParams.start ? queryParams.start : queryParams && queryParams.hashtag ? meta.result.min_timestamp : tempStart,
+                    //     end: queryParams && queryParams.end ? queryParams.end : this.maxDate,
+                    //     countries: queryParams && queryParams.countries ? queryParams.countries : '',
+                    //     topics: queryParams && queryParams.topics ? queryParams.topics : ''
+                    // }
+                    // if (queryParams?.fit_to_content !== undefined) {
+                    //     defaults.fit_to_content = queryParams.fit_to_content
+                    // }
+                    // this.updateURL(defaults)
```

Comment:
same question here, if it does not break anything, we can just remove it right? But are you sure it is unneeded?